src/themes/instagram/views/card.overrides

```diff
@@ -3,7 +3,7 @@
 *******************************/
 
 
-@import url(https://fonts.googleapis.com/css?family=Montserrat:700,400);
+@import (css) url('https://fonts.googleapis.com/css2?family=Montserrat');
```

Comment:
Forgot [weight range](https://developers.google.com/fonts/docs/css2#axis_ranges) ? 
```suggestion
@import (css) url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700,400');
```

Reply:
Same here: only 400 is available and your suggestion results in an unresolved link
https://fonts.googleapis.com/css2?family=Montserrat:wght@700,400
![image](https://user-images.githubusercontent.com/18379884/146695480-245e68b4-d135-4a50-900c-e225b2fcb8eb.png)